tests/tensorflow/mlflow-128-tf-23-env.yaml

```diff
@@ -0,0 +1,11 @@
+channels:
+- conda-forge
+dependencies:
+- python=3.8.12
+- pip<=21.2.4
+- pip:
+  - mlflow==1.28.0
+  - h5py<3.0.0
+  - tensorflow==2.3.0
+  - pandas==1.4.0
```

Comment:
Why do we need pandas 1.4.0?

Reply:
Without this, pickle pandas dataframe failed. See error https://github.com/mlflow/mlflow/actions/runs/3049468798/jobs/4915949671#step:13:124

Reply:
Ok, please add a comment.